docs/themes/oso-tailwind/layouts/partials/sidebar.html

```diff
@@ -32,6 +32,11 @@
                 </div>
                 {{ $currentNode := . }}
                 <ul>
+                    <li class="pl-2 font-bold text-black"><a href="">
+                        <a class="flex items-center px-3 hover:text-gray-900 transition-colors duration-200" href="{{ $.Site.BaseURL }}index.html">
```

Comment:
I'm not sure what this transition is supposed to look like, but I'm not seeing much when I hover on 'Home' in the sidebar

Reply:
I just copied styles from below

Reply:
removed these